test/TestUtil.ts

```
@@ -44,59 +46,48 @@ export class TestUtil {
-    versions = semver.sort(versions);
-    const sqls = await Promise.all(versions.map(version => {
-      return fs.readFile(path.join(__dirname, '../sql', `${version}.sql`), 'utf8');
-    }));
-    return sqls.join('\n');
-  }
-
   static async query(sql: string): Promise<any[]> {
     const conn = this.getConnection();
     return new Promise((resolve, reject) => {
-      conn.query(sql, (err: Error, rows: any[]) => {
+      conn.query(sql, (err: Error, rows: any) => {
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)